plan/task.go

```diff
@@ -465,9 +465,6 @@ func (p *PhysicalHashAgg) newPartialAggregate() (partialAgg, finalAgg *PhysicalH
 	}
 	partialAgg = p.Copy().(*PhysicalHashAgg)
 	// TODO: It's toooooo ugly here. Refactor in the future !!
```

Comment:
Does this TODO still have meaning?

Reply:
the todo message is too abstract here, how about making it more specific ?